WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 110.xcdatamodel</string>
+	<string>WordPress 111.xcdatamodel</string>
```

Comment:
<img width="1009" alt="Screen Shot 2021-01-14 at 3 11 03 PM" src="https://user-images.githubusercontent.com/1435271/104649740-c8375380-567a-11eb-9384-3f60732ac0e1.png">
What do you think about making this a scalar value, providing a default, and making it non-optional?

Reply:
I thought about that, specifically providing a default value. But I didn't want to potentially have a conflict with Calypso. How do we know if a default of seen or unseen is correct?

Reply:
I suppose we don't, but I'm not sure if it matters?  We would want to set it to whatever value was returned when syncing a post.  Have a scalar, non-optional, default is more of a data integrity thing--avoiding nil values that make implicitly unwrapped optionals go :boom:   So, in the cases of a faulted and deleted ReaderPost, if for some reason the object was still being held in memory, if the isSeen field was inspected it would return a boolean rather than nil. 

It's also entirely possible I'm being overly paranoid about it.  CoreData has broken me.